packages/lodestar-cli/src/commands/dev.ts

```diff
@@ -114,20 +115,23 @@ export class DevCommand implements ICliCommand {
         }
         for (let i = start; i < end; i++) {
           const validatorPath = `./validators/validator-db-${i}`;
-          if(fs.existsSync(validatorPath)) {
+          if (fs.existsSync(validatorPath)) {
             rmDir(validatorPath);
           }
         }
       }
     }
 
-    let peerId;
+    let peerId: PeerId;
     if (options["peerId"]) {
       peerId = PeerId.createFromHexString(options["peerId"]);
     } else if (options["peerIdFile"]) {
-      peerId = loadPeerId(options["peerId"]);
+      peerId = await loadPeerId(options["peerId"]);
     } else {
-      peerId = createPeerId();
+      peerId = await createPeerId();
+    }
+    if (!conf.db) {
+      conf.db = {name: join(".", "lodestar-db", peerId.toB58String())};
```

Comment:
This is great, but if everyone agrees, I would maybe propose convention to keep this stuff in `.tmp` directory which we would put in gitignore. This `.tmp` dir would contain all those databases(node, validators, keys etc) as my git status is getting clouded with all those stuff.

cc @wemeetagain @tuyennhv

Reply:
yeah I'm fine with that, let's put these stuff in `.tmp`

Reply:
Good idea!